msteams-platform/bots/streaming-ux.md

```diff
@@ -0,0 +1,553 @@
+---
+title: Stream bot messages
+description: Learn how to enhance the user experience in bots using streaming techniques.
+ms.date: 10/23/2024
+ms.topic: conceptual
+author: surbhigupta12
+ms.author: surbhigupta
+ms.localizationpriority: high
+---
+
+# Stream bot messages
+
+>[!NOTE]
+>
+> - Streaming bot messages is available only for one-on-one chats and in [public developer preview](../resources/dev-preview/developer-preview-intro.md).
+> - Streaming bot messages isn't available with Tools and the `o1` model.
```

Comment:
What tools are not supported, if we can mention?

what is o1 ?
![image](https://github.com/user-attachments/assets/f2d600dd-cbbd-4b08-bcd2-a127cc858b95)

Reply:
Updated input.